### PR TITLE
Code Quality: Refactor IncreaseLayoutSize

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- Refactor IncreaseLayoutSize to prepare for #10492

**Validated**
- Confirmed that adaptive layout is working correctly
- Confirmed that you can use ctrl ++ and ctrl +- to switch between layouts
- Confirmed that you can override layouts

### CODE REVIEW

**Comment by ferrariofilippo** [comment]
I found a bug:
1. Set GridView
2. Increase to max size
3. Set a different layout (Details/List/Tiles)
4. Click on `Small Grid`
5. See that `Large Grid` is selected


https://github.com/files-community/Files/assets/102259289/bfa41b5d-fe85-459c-8e0a-8605ffe22023

**Comment by yaira2** [comment]
Yeah, I've been trying to find a fix but so far I can't find the cause.

**Comment by yaira2** [comment]
That issue should be resolved but there is a different bug where it doesn't work to switch away from columns.

**Comment by yaira2** [comment]
That should be fixed now

File: src/Files.App/Helpers/Layout/LayoutPreferencesManager.cs [hunk 1]
```diff
@@ -40,121 +40,115 @@ public bool IsAdaptiveLayoutEnabled
...
+						break;
+					case FolderLayoutModes.TilesView:
+						SetProperty(ref LayoutPreferencesItem.IconSizeTilesView, value, nameof(IconSize));
+						break;
+					case FolderLayoutModes.GridView:
+						SetProperty(ref LayoutPreferencesItem.IconSizeGridView, value, nameof(IconSize));
+						break;
+					case FolderLayoutModes.ColumnView:
+						SetProperty(ref LayoutPreferencesItem.IconSizeColumnsView, value, nameof(IconSize));
+						break;
```
Comment (0x5bfa) [inline src/Files.App/Helpers/Layout/LayoutPreferencesManager.cs:78]: You could convert this to switch expression.

**Review by 0x5bfa — APPROVED** [review]
Looks good to me as in codebase quality (not tested on the local).

Below is just an idea; depends on you if you change

File: src/Files.App/Helpers/Layout/LayoutPreferencesManager.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Helpers/Layout/LayoutPreferencesManager.cs:78]: Let's leave as is for now

**Comment by yaira2** [comment]
I'm noticing some strange behaviors, but I haven't pinpointed them yet.

**Review by ferrariofilippo — APPROVED** [review]
LGTM and works for me

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**